spec/unit/controllers/services/validators/service_update_validator_spec.rb

```diff
@@ -22,6 +22,8 @@ module VCAP::CloudController
       end
 
       context 'when the update to the service instance is valid' do
+        let(:update_attrs) { { 'service_plan_guid' => new_service_plan.guid } }
```

Comment:
do we need this line? test passes without it.

Reply:
This was added to test the path when we update the plan, and all the checks kick in. We've now added a separate context now in the valid path to check for both scenarios.